kaitai_utils.py

```diff
@@ -45,6 +46,28 @@ def _get_metadata():
         return yaml.safe_load(file)['meta']
 
 
+def object_has_process_key(searchable_object):
```

Comment:
I think this method is only used in this class, right?
```suggestion
def _object_has_process_key(searchable_object):
```

Reply:
done!